### ISSUE TITLE
Documentation is confusing

### ISSUE THREAD

**Issue by stevenbrix** [dotnet/AspNetCore.Docs#19767]
publish profiles aren't only for aspnet.core projects, they apply to Windows desktop apps (WinForms, WPF, WinU)

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 684d1f55-3d8b-9ba6-ee65-e8f84c2b4244
* Version Independent ID: 2db330b4-e830-715f-0a15-e406f85e1d3b
* Content: [Visual Studio publish profiles (.pubxml) for ASP.NET Core app deployment](https://docs.microsoft.com/en-us/aspnet/core/host-and-deploy/visual-studio-publish-profiles?view=aspnetcore-3.1)
* Content Source: [aspnetcore/host-and-deploy/visual-studio-publish-profiles.md](https://github.com/dotnet/AspNetCore.Docs/blob/master/aspnetcore/host-and-deploy/visual-studio-publish-profiles.md)
* Product: **aspnet-core**
* Technology: **aspnetcore-hostdeploy**
* GitHub Login: @Rick-Anderson
* Microsoft Alias: **riande**

**Comment by Rick-Anderson** [comment]
@stevenbrix  true, but the title is ***Visual Studio publish profiles (.pubxml) for ASP.NET Core app deployment**

Do you want to PR this and add that info?

**Comment by stevenbrix** [comment]
@Rick-Anderson That's fair. The confusing part is that you google publish profiles and this is the only thing that shows up.

Is there a different place this could doc could move to?

**Comment by Rick-Anderson** [comment]
We could move this to https://github.com/MicrosoftDocs/visualstudio-docs-pr and you, @sayedihashimi, @TerryGLee, or @ghogen could add a section for WinForms, WPF, WInU, etc.

**Comment by ghogen** [comment]
@stevenbrix @Rick-Anderson I could improve our docs on publish profiles in the VS docs, so yes.

**Comment by sayedihashimi** [comment]
Wow, I never knew that windows projects now use the publish profile format. That is really great news. I will have to check that out sometime.

**Comment by stevenbrix** [comment]
Thank you all for being so responsive and on top of things, you're seriously awesome!
